src/components/shared/Dropdown.css

```diff
@@ -39,6 +35,15 @@ html[dir="rtl"] .dropdown {
   width: 24px;
 }
 
+.moreTabs {
```

Comment:
The convention in css is to use `-` instead of camel case, don't know how it slipped before. But this should be the right time to fix it :)

Reply:
Thanks for the suggestion and your time @AnshulMalik 

implementing now.